encryption_works.md

```diff
@@ -168,7 +168,7 @@ Encrypting your hard-drive doesn't just make it harder for the NSA to access you
 
 Disk encryption is available for Mac OS X, Windows, and Linux, but the steps for enabling it differ for each operating system. For information on how to encrypt your disk, check out Micah Lee's guide to [encrypting your laptop like you mean it](https://firstlook.org/theintercept/2015/04/27/encrypting-laptop-like-mean/) on The Intercept.
 
-For journalists who travel internationally, it's important to remember that federal courts have traditionally [affirmed](https://www.eff.org/files/eff-border-search_2.pdf) that the government does not need suspicion of criminal activity to search a traveler's laptop at the border (the ninth circuit being the only [exception](https://scholar.google.com/scholar_case?case=13727924395632289367)). Non-citizens may be turned away if they do not comply, and U.S. citizens can face extended questioning. It's up to the individual to decide how much hassle they're willing to risk, but there are some steps you can take to secure files when traveling. 
+For journalists who travel internationally, it's important to remember that federal courts have traditionally [affirmed](https://www.eff.org/files/eff-border-search_2.pdf) that the government does not need suspicion of criminal activity to search a traveler's laptop at the border (the ninth circuit being the only [exception](https://scholar.google.com/scholar_case?case=13727924395632289367)). Non-citizens may be turned away if they do not comply, and US citizens can face extended questioning. It's up to the individual to decide how much hassle they're willing to risk, but there are some steps you can take to secure files when traveling. 
```

Comment:
can we find a non-pdf link for the boarder search stuff?  I know it's from EFF and we love them, but I hate pointing people to pdfs.